### ISSUE TITLE
ERROR: Package path ./data is not exported from package

### ISSUE THREAD

**Issue by 6uzm4n** [mledoze/countries#484]
After updating with npm to version 4.1+ the following error occurs in my project:

`
Package path ./data is not exported from package ~/myapp/node_modules/world-countries (see exports field in ~/myapp/node_modules/world-countries/package.json)
`

This is something that doesn't happen in version 4.0.0

**Comment by mledoze** [comment]
Here is what I have:

```
$ npm -v
9.8.1

$ npm install world-countries

added 1 package in 3s

$ npm list
test
└── world-countries@4.1.1
```

Could you describe how to reproduce the error?

**Comment by mledoze** [comment]
fixed by #486, thank you @thevuong
